applications/asset_tracker/Kconfig

```diff
@@ -57,6 +57,14 @@ config GPS_SIM_TRIGGER_TIMER_MSEC
 	default 120000 if POWER_OPTIMIZATION_ENABLE
 	default 2000
 
+config GPS_START_ON_MOTION
+	bool "Motion triggered GPS update"
+	default n
+	help
+		When accelerometer detects movement, and GPS is enabled,
+		update GPS position, but if no movement, wait a longer
+		delay between updates to conserve power.
```

Comment:
The whitespace doesn't look correct here. Please compare with other help texts.

Reply:
The help text above and below that has the same format -- one tab for the 'help' line and two tabs for the indented help text.  Or did you mean something else?

The 'endmenu # GPS' line below my new section does have a tab unlike other sections which have a single space, so I can correct that if you wish.

Further down in the file, there are older help text sections that are instead one tab plus two spaces, and another section which is one tab plus four spaces.  So the file itself is not uniform.

Do you want me to fix all the formatting to be uniform, despite it being unrelated to my PR?

Reply:
This is a known bug in Zephyr:
https://github.com/zephyrproject-rtos/zephyr/issues/18351
https://github.com/zephyrproject-rtos/zephyr/pull/18921

The solution seems to be what I actually did myself.

Reply:
Hi,

the correct formatting for the help text is one tab and two spaces.

Please fix for your patch. I leave it up to you if you wish to fix the other help texts or not.

Reply:
Also, you accidentally added a comment about logging floating point here.

Reply:
Moved my logging comment to the correct section.

Fixed the whitespace on my main commit.  Added a second cleanup commit for all other help text in application/asset_tracker/Kconfig.

Reply:
By the way, @SebastianBoe can you point me to a document covering the Kconfig formatting rules?  I looked in NCS, Zephyr, and Linux Kernel documents about Kconfig and have not found anything that mentions the tabs and/or spaces required specifically.

Reply:
I was not able to find this specified. PR posted:

https://github.com/zephyrproject-rtos/zephyr/pull/22969